modules/overrides/user_configurable_overrides.go

```diff
@@ -215,6 +215,10 @@ func (o *userConfigurableOverridesManager) MetricsGeneratorDisableCollection(use
 	return o.Interface.MetricsGeneratorDisableCollection(userID)
 }
 
+func (o *userConfigurableOverridesManager) MetricsGenerationTraceIDLabelName(userID string) string {
+	return o.Interface.MetricsGenerationTraceIDLabelName(userID)
+}
+
```

Comment:
If we don't want to expose this setting through the API we can just omit this function completely.
`Interface` is a nested struct of `userConfigurableOverridesManager` so if we don't implement this function Go will automatically call the implementation on the nest struct (`Interface`).

Reply:
thanks - removed